src/fsharp/FSharp.Core/async.fsi

```diff
@@ -299,8 +532,49 @@ namespace Microsoft.FSharp.Control
         /// <returns>A computation that returns the first succeeding computation.</returns>
         ///
         /// <category index="1">Composing Async Computations</category>
-        /// 
-        /// <example-tbd></example-tbd>
+        ///
+        /// <example id="choice-example-1">
+        /// <code lang="fsharp">
+        /// printfn "Starting"
+        /// [ 2; 3; 5; 7 ]
+        /// |> List.map
+        ///     (fun i ->
+        ///         async {
+        ///             do! Async.Sleep(System.Random().Next(1000, 2000))
+        ///             return if i % 2 > 0 then Some(i) else None
+        ///         })
+        /// |> Async.Choice
+        /// |> Async.RunSynchronously
+        /// |> function
```

Comment:
Generally we don't use "pipe into fun" or "pipe into function" in beginner samples - since it's just another way of doing a `let` binding then a match.